web/Views/Property/PropertyInputForm/TenantPreferences.cshtml

```diff
@@ -7,15 +7,15 @@
 
     <div class="row g-3">
         <div class="col-auto d-grid">
-            @Html.CheckBox("AcceptsSingleTenant", Model.AcceptsSingleTenant ?? false, new { @class = "btn-check" })
-            <label asp-for="AcceptsSingleTenant" class="btn btn-outline-blue shadow-none pill-check">Single tenant</label>
+            <input type="checkbox" asp-for="AcceptsSingleTenant" class="btn-check"/>
+            <label asp-for="AcceptsSingleTenant" id="AcceptsSingleTenant" class="btn btn-outline-blue shadow-none pill-check">Single tenant</label>
         </div>
         <div class="col-auto d-grid">
-            @Html.CheckBox("AcceptsCouple", Model.AcceptsCouple ?? false, new { @class = "btn-check" })
+            <input type="checkbox" asp-for="AcceptsCouple" class="btn-check"/>
             <label asp-for="AcceptsCouple" class="btn btn-outline-blue shadow-none pill-check">Couple</label>

```

Comment:
Should these labels gain IDs too?

Reply:
The labels are the buttons, so the asp-for is intended to give them the id and name to allow the user to click it and change the underlying boolean. The input checkbox is hidden, but linked to the same thing, allowing the CSS to look for "input:checked ~ label" and show the label button as pressed.

Reply:
In which case should we remove the ID from the button above?

Reply:
Yes! :) fixed